packages/js/components/src/product-fields/stories/index.tsx

```diff
@@ -52,7 +38,18 @@ const RenderField = () => {
 					</option>
 				) ) }
 			</select>
-			{ selectedField && renderField( selectedField, { name: 'test' } ) }
+			{ selectedField &&
```

Comment:
Optional thought. It looks like you're rendering all fields with the same config, some of which aren't needed for all fields. This works fine, but as this is a story it might be clearer to be more explicit when rendering fields with the expected config. Perhaps just adding another quick story to show this, such as the toggle with a tooltip would work.